test/UserTest.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (C) 2004-2015 ZNC, see the NOTICE file for details.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+#include <znc/User.h>
+#include <znc/znc.h>
+
+class UserTest : public ::testing::Test {
+  protected:
+    UserTest() { CZNC::CreateInstance(); }
+    ~UserTest() { CZNC::DestroyInstance(); }
+};
```

Comment:
If I remove the CZNC calls, it looks like the unit test segfaults when instantiating a CUser.